### ISSUE TITLE
Remove examples using cardano-wallet integrations

### ISSUE THREAD

**Issue by rdlrt** [cardano-foundation/developer-portal#853]
I think `cardano-wallet` is a nice (easy) use-case for individuals running full node and for small-mid custodial services (like exchanges).

But as I keep mentioning on issues/PR-reviews, it's not suitable for service integrations (it would have been relevant perhaps 2 years ago).

On developers.cardano.org itself , we have a few examples that are relevant (still part of above) using cardano-wallet, but users may naively use them thinking of it as a recommended method. We should reduce the coverage and get it replaced with tools that work better for integrations (for example, a mix of cardano-[serialisation|multiplatform]-lib/lucid and ogmios/kupo, scrolls/oura, or dbsync/graphql/koios to interact with chain instead).

**Comment by rphair** [comment]
I agree & think this is also an invitation to build systems vulnerable to hot wallet hacks (if not carefully designed) which are getting more bad press recently ([high profile example](https://www.theblock.co/post/182048/crypto-exchange-deribit-loses-28-million-in-hack-halts-withdrawals)).

**Comment by Mercurial** [comment]
I also agree with this, when we started the developer portal, the mindset was to set the most simplest example possible to covet he concepts. I do believe it's time to step it up and that the dev portal endorse a best practice while giving good proof of concept examples at the same time. This was also my sentiment with other sections during the time when the developer portal was newly launched.
